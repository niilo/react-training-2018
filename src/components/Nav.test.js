import React from 'react';
import {BrowserRouter, Route} from 'react-router-dom';
import renderer from 'react-test-renderer';
import Nav from './Nav';

it('renders default correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Route>
          <Nav />
        </Route>
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
