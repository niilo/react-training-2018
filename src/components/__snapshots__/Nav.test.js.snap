// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default correctly 1`] = `
<ul
  className="nav"
>
  <li>
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Home
    </a>
  </li>
  <li>
    <a
      aria-current={null}
      className={undefined}
      href="/battle"
      onClick={[Function]}
      style={undefined}
    >
      Battle
    </a>
  </li>
  <li>
    <a
      aria-current={null}
      className={undefined}
      href="/popular"
      onClick={[Function]}
      style={undefined}
    >
      Popular
    </a>
  </li>
</ul>
`;
